media-sound/audacity/audacity-3.2.2.ebuild

```diff
@@ -0,0 +1,221 @@
+#   supported at runtime.  See: libraries/lib-ffmpeg-support/impl
+#   Current support grid:
+#   - Lavf - 5[789]
+#   - Lavc - 5[789]
+#   - Lavu - 5[2567]
+
+RDEPEND="dev-db/sqlite:3
+	dev-libs/expat
+	dev-libs/glib:2
+	media-libs/libsndfile
+	media-libs/libsoundtouch:=
+	media-libs/portaudio[alsa?]
+	media-libs/portmidi
+	media-libs/portsmf:=
+	media-libs/soxr
+	>=media-sound/lame-3.100-r3
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:3
+	x11-libs/wxGTK:${WX_GTK_VER}[X]
+	sys-apps/util-linux
+	alsa? ( media-libs/alsa-lib )
+	audiocom? (
+		dev-libs/rapidjson
+		net-misc/curl
+	)
+	ffmpeg? ( media-video/ffmpeg )
+	flac? ( media-libs/flac:=[cxx] )
+	id3tag? ( media-libs/libid3tag:= )
+	lv2? (
+		dev-libs/serd
+		dev-libs/sord
+		>=media-libs/lilv-0.24.6-r2
+		media-libs/lv2
+		media-libs/sratom
+		media-libs/suil
+	)
+	mad? ( >=media-libs/libmad-0.15.1b )
+	mpg123? ( media-sound/mpg123 )
+	ogg? ( media-libs/libogg )
+	opus? ( media-libs/opus )
+	sbsms? ( media-libs/libsbsms )
+	twolame? ( media-sound/twolame )
+	vamp? ( media-libs/vamp-plugin-sdk )
+	vorbis? ( media-libs/libvorbis )
+	wavpack? ( media-sound/wavpack )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="app-arch/unzip
+	sys-devel/gettext
+	virtual/pkgconfig
+"
+
+REQUIRED_USE="
+	mad? ( !mpg123 )
+"
+
+PATCHES=(
+	# Equivalent to previous versions
+	"${FILESDIR}/${PN}-3.2.2-disable-ccache.patch"
+	# From Debian
+	"${FILESDIR}/${PN}-3.2.2-fix-rpaths.patch"
+
+	# Disables some header-based detection
+	"${FILESDIR}/${PN}-3.2.2-allow-overriding-alsa-jack.patch"
+
+	# For has_networking
+	"${FILESDIR}/${PN}-3.2.2-remove-conan-threadpool.patch"
+	"${FILESDIR}/${PN}-3.2.2-allow-finding-rapidjson-pkgconfig.patch"
+)
+
+src_prepare() {
+	cmake_src_prepare
+
+	local header_subs="${S}/lib-src/header-substitutes"
+	cat <<-EOF >"${header_subs}/allegro.h" || die
+	/* Hack the allegro.h header substitute to use system headers.  */
+	#include <portsmf/allegro.h>
+	EOF
+
+	# Keep in sync with has_networking and the ThreadPool.h SRC_URI.
+	if use audiocom; then
+		mkdir -p "${S}/"/lib-src/threadpool/ThreadPool/ || die
+		cp "${DISTDIR}"/progschj-ThreadPool-"${MY_THREADPOOL_DATE}".h \
+		   "${S}"/lib-src/threadpool/ThreadPool/ThreadPool.h || die
+	fi
+}
+
+src_configure() {
+	setup-wxwidgets
+
+	# * always use system libraries if possible
+	# * options listed roughly in the order specified in
+	#   cmake-proxies/CMakeLists.txt
+	# * USE_VST was omitted, it appears to no longer have dependencies
+	#   (this is different from VST3)
+	local mycmakeargs=(
+		-Daudacity_use_nyquist=local
+		#-Daudacity_use_pch leaving it to the default behavior
+		-Daudacity_use_portmixer=$(usex portmixer system off)
+		-Daudacity_use_soxr=system
+
+		-Daudacity_conan_enabled=OFF
+
+		-Daudacity_has_networking=$(usex audiocom on off)
+		# Not useful on Gentoo.
+		-Daudacity_has_updates_check=OFF
+		-Daudacity_has_audiocom_upload=$(usex audiocom on off)
+		-Daudacity_has_threadpool=local
```

Comment:
Then I'm getting:
```
 * One or more CMake variables were not used by the project:
 *   audacity_has_threadpool
```
with:
USE:        abi_x86_64 alsa amd64 elibc_glibc ffmpeg kernel_linux ladspa mad ogg userland_GNU wavpack

USE:        abi_x86_64 amd64 elibc_glibc flac kernel_linux ladspa lv2 portmixer userland_GNU vorbis

USE:        abi_x86_64 amd64 doc elibc_glibc flac id3tag kernel_linux mpg123 sbsms userland_GNU vamp vorbis wavpack

Could you try to dig out the logic for these? Although I'm not 100 % sure if toralf and ago report these cmake issues anymore, fixing these may not seem important.